hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/ReactiveConnectionPool.java

```diff
@@ -31,8 +31,16 @@
 public interface ReactiveConnectionPool extends Service {
 
 	/**
-	 * Obtain a reactive connection.
+	 * Obtain a reactive connection, retuning the connection
```

Comment:
Typo: returning

Reply:
Yeah, but c'mon, it would be cool if it did "retune" things.